runtime/doc/lua.txt

```diff
@@ -841,10 +841,18 @@ vim.call({func}, {...})					*vim.call()*
             vim.fn[func]({...})
 
 vim.cmd({cmd})                                          *vim.cmd()*
-        Invokes an Ex command (the ":" commands, Vimscript statements).
+        Executes multiple lines of Vimscript at once. It is an alias to
+        |nvim_exec()|, where `output` is set to false. Thus it works identical
+        to `|:source|`.
```

Comment:
One last thing (my bad, I wrote it in my comment), but the backticks are wrong here. Should be just `|:source|`.

Reply:
fixed